webui/react/src/e2e/models/pages/WorkspaceList.ts

```diff
@@ -4,10 +4,26 @@ import { DeterminedPage } from 'e2e/models/common/base/BasePage';
 import { Message } from 'e2e/models/common/hew/Message';
 import { Select } from 'e2e/models/common/hew/Select';
 import { Toggle } from 'e2e/models/common/hew/Toggle';
+import { GridListRadioGroup } from 'e2e/models/components/GridListRadioGroup';
+import { HeadRow, InteractiveTable, Row } from 'e2e/models/components/Table/InteractiveTable';
 import { WorkspaceCreateModal } from 'e2e/models/components/WorkspaceCreateModal';
 import { WorkspaceDeleteModal } from 'e2e/models/components/WorkspaceDeleteModal';
 import { WorkspaceCard } from 'e2e/models/pages/WorkspaceList/WorkspaceCard';
 
+class WorkspaceHeadRow extends HeadRow {
+  readonly name = new BaseComponent({
+    parent: this,
+    selector: '[data-testid="name"]',
+  });
+}
+
+class WorkspaceRow extends Row {
+  readonly name = new BaseComponent({
+    parent: this,
+    selector: '[data-testid="name"]',
```

Comment:
I like to use the single approach because it makes for good references. The approach I chose comes from my experience, but if perhaps Hew is expecting something else, then it's okay to have exceptions.